scribus/plugins/tools/hunspellcheck/hunspellpluginimpl.cpp

```diff
@@ -181,7 +181,7 @@ bool HunspellPluginImpl::parseTextFrame(StoryText *iText)
 			wf.replacements = hspellerMap[wordLang]->suggest(word);
 			wordsToCorrect.append(wf);
 		}
-		currPos = iText->nextWord(wordStart);
+		currPos = iText->endOfWord(wordStart);
```

Comment:
If this is the only place this was calling `nextWord()`, then keep this change and just kill `nextWord()` and `prevWord()` since both will be unused now.